specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-11-01/examples/createComputeInstance.json

```diff
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "computeName": "compute123",
+    "api-version": "2019-11-01",
+    "parameters": {
+      "location": "eastus",
+      "properties": {
+        "computeType": "ComputeInstance",
+        "properties": {
+          "vmSize": "STANDARD_NC6",
+          "subnet": "test-subnet-resource-id",
+          "applicationSharingPolicy": "Personal",
```

Comment:
I didn't see this defined in the amlInstance request and response. Do you miss that in the definition?

Reply:
Yu, as discussed, you were not looking at the latest version.